sources/DIRECTORY

```diff
@@ -95,17 +101,26 @@ Copyright (c) 1986, 1987, 1988, 1990, 1992 by Venue & Xerox Corporation.  All ri
                              then (push DESIREDPROPS 'IREADDATE))
                          (RPLACA (SETQ COMTAIL (CDR COMTAIL))
                                 (if (NUMBERP (SETQ COM (CAR COMTAIL)))
-                                    then                 (* ; "A number of days")
-                                          [IDIFFERENCE (IDATE)
-                                                 (TIMES COM (DEFERREDCONSTANT (IDIFFERENCE
-                                                                               (IDATE 
-                                                                                     "2-JAN-77 00:00"
-                                                                                      )
-                                                                               (IDATE 
-                                                                                     "1-JAN-77 00:00"
-                                                                                      ]
+                                    then                     (* ; "A number of days")
+                                         [IDIFFERENCE (IDATE)
+                                                (TIMES COM (DEFERREDCONSTANT (IDIFFERENCE
+                                                                              (IDATE "2-JAN-77 00:00"
+                                                                                     )
+                                                                              (IDATE "1-JAN-77 00:00"
+                                                                                     ]
                                   elseif (IDATE COM)
                                   else (\ILLEGAL.ARG COM))))
+                    (DEPTH [SETQ FILING.ENUMERATION.DEPTH (IF (AND (SMALLP (CADR COMTAIL))
```

Comment:
I think this isn't enough to work well -- probably need to make FILING.ENUMERATION.DEPTH special and rebind it rather than resetting it globally.